doc/services/pm/device.rst

```diff
@@ -10,37 +10,68 @@ setting :kconfig:option:`CONFIG_PM_DEVICE` to ``y``. When this option is
-and has the best knowledge of their use, performs the device power management. This
-saves power if some devices that are not in use can be turned off or put
-in power saving mode. This method allows saving power even when the CPU is
-active. The components that use the devices need to be power aware and should
-be able to make decisions related to managing device power.
-
-When using this type of device power management, the kernel can change CPU
-power states quickly when :c:func:`pm_system_suspend()` gets called. This is
-because it does not need to spend time doing device power management if the
-devices are already put in the appropriate power state by the application or
-component managing the devices.
+Device runtime power management involves coordinated interaction between
+device drivers, subsystems, and applications. While device drivers
+play a crucial role in directly controlling the power state of
+devices, the decision to suspend or resume a device can also be
+influenced by higher layers of the software stack.
+
+Each layer—device drivers, subsystems, and applications—can operate
+independently without needing to know about the specifics of the other
+layers because the subsystem uses reference count to check when it needs
+to suspend or resume a device.
+
+ - **Device drivers** are responsible for managing the
+   power state of devices. They interact directly with the hardware to
+   put devices into low-power states (suspend) when they are not in
+   use, and bring them back (resume) when needed. Drivers should use the
+   :ref:`device runtime power management APIs <device_runtime_apis>` provided
+   by Zephyr to control the power state of devices.
+
+ - **Subsystems**, such as sensors, file systems,
+   and network, can also influence device power management.
+   Subsystems may have better knowledge about the overall system
+   state and workload, allowing them to make informed decisions about
+   when to suspend or resume devices. For example, a networking
+   subsystem may decide to keep a network interface powered on if it
+   expects network activity in the near future.
+
+ - **Applications** running on Zephyr can impact device
+   power management as well. An application may have specific
+   requirements regarding device usage and power consumption. For
+   example, an application that streams data over a network may need
+   to keep the network interface powered on continuously.
+
+Coordination between device drivers, subsystems, and applications is
+key to efficient device power management. For example, a device driver
+may not know that a subsystem will perform a series of sequential
+operations that require a device to remain powered on. In such cases,
+the subsystem can use device runtime power management to ensure that
+the device remains in an active state until the operations are
+complete.
+
+When using this Device Runtime Power Management, the Power Management subsystem
```

Comment:
This is a bit clearer to me:
`When using Device Runtime Power Management, the System Power Management subsystem is able to change system power states quickly...`

Reply:
sounds better indeed